db/migrate/20230120144741_create_user.rb

```diff
@@ -1,12 +1,14 @@
 class CreateUser < ActiveRecord::Migration[7.0]
-  def change
-    create_table :users do |t|
-      t.string :Name
-      t.string :Photo
-      t.text :Bio
-      t.integer :PostsCounter
+   def change
+      create_table :users do |t|
+         t.string :Name
+         t.string :Photo
+         t.text :Bio
+         t.integer :PostsCounter
+         t.datetime :UpdatedAt
+         t.datetime :CreatedAt
 
-      t.timestamps
-    end
-  end
+         t.timestamps
```

Comment:
- [ ] `t.timestams` will crate updated_at and created_at columns, which means that you don't need to create extra columns, please 🙏🏽  remove UpdateAt and CreateAt columns to avoid future conflicts.